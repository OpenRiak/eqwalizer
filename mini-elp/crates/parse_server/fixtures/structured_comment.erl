%%% Copyright (c) Meta Platforms, Inc. and affiliates. All rights reserved.
%%%
%%% This source code is licensed under the Apache 2.0 license found in
%%% the LICENSE file in the root directory of this source tree.

-module(structured_comment).

test(X) ->
    %$eqwalizer: dynamic(X)
    X.

test2(X) ->
    %$eqwalizer: X :: dynamic()
    X.
