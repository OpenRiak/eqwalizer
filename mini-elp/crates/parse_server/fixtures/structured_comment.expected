{attribute,{0,0},file,{"fixtures/structured_comment.erl",0}}.

{attribute,{216,243},module,structured_comment}.

{function,{246,290},
          test,1,
          [{clause,{246,290},[{var,{251,252},'X'}],[],[{var,{289,290},'X'}]}]}.

{function,{293,342},
          test2,1,
          [{clause,{293,342},[{var,{299,300},'X'}],[],[{var,{341,342},'X'}]}]}.

{eof,{344,344}}.
